tests/regression/common.py

```diff
@@ -5,8 +5,9 @@
 import requests
 import statistics
 import subprocess
+import inspect
+import argparse
```

Comment:
Add these modules to `requirements.txt`

Reply:
I'll update the requirements from the output of `pip freeze`